frontend/src/ui/player/index.tsx

```diff
@@ -118,16 +118,38 @@ const delayTill = (date: Date): number => {
 export const InlinePlayer: React.FC<PlayerProps> = ({ className, event, ...playerProps }) => {
     const aspectRatio = getPlayerAspectRatio(event.syncedData.tracks);
     const isDark = useColorScheme().scheme === "dark";
+    const ref = useRef<HTMLDivElement>(null);
+
+    const minControlWidth = 470;
+    const [controlsFit, setControlsFit] = useState<boolean>(true);
+
+    const resizeObserver = new ResizeObserver(() => {
+        if (ref.current) {
+            const scaling = ref.current.offsetHeight / aspectRatio[1];
+            const inferredWidth = scaling * aspectRatio[0];
+            setControlsFit(inferredWidth >= minControlWidth);
+        }
+    });
+
+    useEffect(() => {
+        if (ref.current) {
+            resizeObserver.observe(ref.current);
+        }
+        return () => resizeObserver.disconnect();
+    });
 
     return (
-        <div className={className} css={{
-            "--video-container-background-color": COLORS.neutral10,
-            "--base-video-rect-background-color": COLORS.neutral10,
+        <div {...{ className }} {...{ ref }} css={{
+            "--video-container-background-color": "#101010",
+            "--base-video-rect-background-color": "#101010",
```

Comment:
While I do think `#101010` looks good, I can't help but raise a voice for the OLED monitor gang: especially in full screen mode, you don't want the letterboxing to be anything but `#000` so that those pixels can be turned off completely. In non-fullscreen it probably doesn't matter too much, I imagine. I just checked and while I first thought this might increase the contrast to the light mode page-background too much, I don't think it makes a large difference. So I think I would change it to `#000`.

Reply:
The reason I chose `#101010` is actually (contrary to my commit message...) that the preview images of portrait videos use that color for their letterboxing. Which I only noticed because I tried `#000` first. It is a subtle but still noticeable difference:
<img width="1179" alt="Bildschirmfoto 2023-11-07 um 10 24 38" src="https://github.com/elan-ev/tobira/assets/94838646/22f4ae80-39c3-450c-a421-fecd357273c6">

But that's not actually done by the variables you commented this on - I only changed those for consistency's sake. They can be `#000` without issues. Would you be ok with only changig these?

Reply:
Oh hu, good call. I think I would still go with `#000`. The fact that the preview image has letterboxing is bad anyway. So I'd say this problem should be fixed on the Opencast workflow side, also for other reasons.